python/ray/serve/_private/router.py

```diff
@@ -229,9 +229,14 @@ async def get_queue_state(self) -> Tuple[int, bool]:
         # NOTE(edoakes): the `get_num_ongoing_requests` method name is shared by
         # the Python and Java replica implementations. If you change it, you need to
         # change both (or introduce a branch here).
-        queue_len = await self._actor_handle.get_num_ongoing_requests.remote()
-        accepted = queue_len < self._replica_info.max_concurrent_queries
-        return queue_len, accepted
+        obj_ref = self._actor_handle.get_num_ongoing_requests.remote()
+        try:
```

Comment:
you mention `gets cancelled if we don't get a response within 100ms`
For my knowledge, where is it configured?

Reply:
It's in `select_from_candidate_replicas` and controlled by `queue_len_response_deadline_s`